core/templates/dev/head/pages/question_editor/QuestionEditorDirective.js

```diff
@@ -105,15 +105,16 @@ oppia.directive('questionEditor', [
           };
 
           $scope.saveInteractionAnswerGroups = function(newAnswerGroups) {
+            for (var i = 0; i < newAnswerGroups.length; i++) {
```

Comment:
No, with this method of not showing the outcome destination editor at all, it is set to same state name (i.e try again) by default, so all of them are changed to null here, since this is really something exclusive to questions.

Reply:
Ok, will try that.

Reply:
Done, setting destination as null in StateResponsesDirective dureing creation itself..